### PR TITLE
Added the Manual pre-requisite steps for avago9361 - v4

### PR DESCRIPTION
**Pull request by manvanthar**
Added the Manual pre-requisite steps for avago9361

Signed-off-by: Manvanthara Puttashankar <manvanth@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by manvanthar** [comment]
# avocado run avago9361_vd.py -m avago9361_vd.py.data/avago9361_vd.yaml 
JOB ID     : ca6442592a35df860c3fd3947765165a7cdadb45
JOB LOG    : /root/tests/results/job-2019-10-18T00.31-ca64425/job.log
 (01/18) avago9361_vd.py:Avago9361.test_createall;Scenario-r0-2af5: PASS (123.51 s)
 (02/18) avago9361_vd.py:Avago9361.test_createall;Scenario-r1-e663: PASS (123.98 s)
 (03/18) avago9361_vd.py:Avago9361.test_maxvd;Scenario-r0-2af5: PASS (204.48 s)
 (04/18) avago9361_vd.py:Avago9361.test_maxvd;Scenario-r1-e663: PASS (155.35 s)
 (05/18) avago9361_vd.py:Avago9361.test_init;Scenario-r0-2af5: PASS (33.25 s)
 (06/18) avago9361_vd.py:Avago9361.test_init;Scenario-r1-e663: PASS (62.94 s)
 (07/18) avago9361_vd.py:Avago9361.test_cc;Scenario-r0-2af5: CANCEL: Test not applicable for Raid0 (0.01 s)
 (08/18) avago9361_vd.py:Avago9361.test_cc;Scenario-r1-e663: PASS (173.21 s)
 (09/18) avago9361_vd.py:Avago9361.test_online_offline;Scenario-r0-2af5: CANCEL: Test not applicable for Raid0 (0.00 s)
 (10/18) avago9361_vd.py:Avago9361.test_online_offline;Scenario-r1-e663: PASS (108.99 s)
 (11/18) avago9361_vd.py:Avago9361.test_ghs_dhs;Scenario-r0-2af5: CANCEL: Test not applicable for Raid0 (0.01 s)
 (12/18) avago9361_vd.py:Avago9361.test_ghs_dhs;Scenario-r1-e663: PASS (977.66 s)
 (13/18) avago9361_vd.py:Avago9361.test_rebuild;Scenario-r0-2af5: CANCEL: Test not applicable for Raid0 (0.00 s)
 (14/18) avago9361_vd.py:Avago9361.test_rebuild;Scenario-r1-e663: PASS (113.33 s)
 (15/18) avago9361_vd.py:Avago9361.test_patrolread;Scenario-r0-2af5: PASS (52.43 s)
 (16/18) avago9361_vd.py:Avago9361.test_patrolread;Scenario-r1-e663: PASS (52.64 s)
 (17/18) avago9361_vd.py:Avago9361.test_migrate;Scenario-r0-2af5: PASS (2.23 s)
 (18/18) avago9361_vd.py:Avago9361.test_migrate;Scenario-r1-e663: CANCEL: Script runs for raid0 (0.01 s)
RESULTS    : PASS 13 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 5
JOB TIME   : 2184.57 s
JOB HTML   : /root/tests/results/job-2019-10-18T00.31-ca64425/results.html

**Comment by manvanthar** [comment]
[job.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/3741048/job.log)

File: io/disk/Avago_storage_adapter/avago9361_vd.py [hunk 1]
```diff
@@ -315,21 +314,23 @@ def vd_create(self, write, read, iopolicy, stripe):
...
+            self.check_pass(cmd, 'False', "Failed to create raid")
         self.vd_details()
 
-    def check_pass(self, cmd, errmsg):
+    def check_pass(self, cmd, ignore_val, errmsg):
         """
         Helper function to check, if the cmd is passed or failed
         """
-        if process.system(cmd, ignore_status=True, shell=True) != 0:
+        if ignore_val == 'True':
```
Comment (narasimhan-v) [inline io/disk/Avago_storage_adapter/avago9361_vd.py]: please make it boolean

**Comment by narasimhan-v** [comment]
@abdhaleegit review please

**Pull request closed without merging**